decoder/vaapidecoder_h264.cpp

```diff
@@ -1760,7 +1760,7 @@ YamiStatus VaapiDecoderH264::decodeSlice(NalUnit* nalu)
         status = createPicture(slice, nalu);
         if (status != YAMI_SUCCESS)
             return status;
-        if (!m_currPic
+        if (!m_currPic || !m_prevPic
```

Comment:
@laimatth , this patch is not to fix the SEI NAL issue, it is to fix the first slice is not IDR slice issue, right? 
1. I think you can refer  https://github.com/01org/libyami/pull/623/commits/4dca2e071159bf9a28581bbc83c56f460667a420 to fix such issue, because actually we don't need Previous Frame if current frame is IDR frame. 
2. Would better to change the commit title and message. It is to fix h264 decoder issue not V4L2 issue. Maybe you can change it to "h264dec: don't start decode unit IDR slice", and create a V4L2 decoder issue then link it to the issue in the commit message.